template/src/redux/rootReducer.ts

```diff
@@ -1,7 +1,25 @@
+import { AnyAction, combineReducers } from 'redux'
+import { persistReducer } from 'redux-persist'
+import authReducer from '@api/authSlice'
 import demoReducer from '@screens/demoSlice'
+import { storage } from './persistence'
+import { resetStore } from './rootActions'
 
-const rootReducer = {
+const appReducer = combineReducers({
+  auth: authReducer,
   demo: demoReducer,
+})
+
+const rootReducer = (state: ReturnType<typeof appReducer>, action: AnyAction) => {
+  if (resetStore.match(action)) {
+    return appReducer(undefined, action)
+  }
+  return appReducer(state, action)
 }
 
-export default rootReducer
+const rootPersistConfig = {
+  key: 'root',
+  storage,
+  blacklist: ['auth'],
```

Comment:
By default we don't want to persist auth tokens? Will users have to sign in again each time app is restarted or do I miss something? Is it what we want?

Reply:
AuthSlice have it's own persist config and it's persisted separately in encrypted storage. If we don't blacklist it here, it's gonna be stored in 2 places at the same time